config/application.rb

```diff
@@ -312,11 +312,9 @@ def watchable_args
       config.relative_url_root = GlobalSetting.relative_url_root
     end
 
-    if Rails.env == "test"
-      if ENV['LOAD_PLUGINS'] == "1"
-        Discourse.activate_plugins!
-      end
-    else
+    if Rails.env == "test" && GlobalSetting.load_plugins?
```

Comment:
Tiny but "consistenter"?

```suggestion
    if Rails.env.test? && GlobalSetting.load_plugins?
```